android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

```diff
@@ -746,6 +746,40 @@ public String getAllResponseHeaders()
 		return result;
 	}
 
+	public KrollDict getResponseHeaders()
+	{
+		final KrollDict result = new KrollDict();
+
+		// Obtain response headers.
+		if (responseHeaders != null && !responseHeaders.isEmpty()) {
+			final Set<Map.Entry<String, List<String>>> entrySet = responseHeaders.entrySet();
+
+			// Iterate through response headers.
+			for (Map.Entry<String, List<String>> entry : entrySet) {
+				final String key = entry.getKey();
```

Comment:
Can you add the following if-check please?
```javascript
final String key = entry.getKey();
if (key == null) {
	continue;
}
```

This is because the Java `HTTPUrlConnection` class always returns a header entry with a `null` key providing the HTTP "status line" (ex: `"HTTP/1.1 200 OK"`). While the 1st line the HTTP response is the status line, it's not considered the header. And I've confirmed that iOS' `HTTPClient.responseHeaders` does not return a null/StatusLine entry making this behavior consistent with iOS.
https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html